specs/whisk/beacon-chain.md

```diff
@@ -169,19 +163,10 @@ def process_epoch(state: BeaconState) -> None:
 #### `BeaconBlock`
 
 ```python
-class BeaconBlock(bellatrix.BeaconBlock):
-    # ...
-    body: BeaconBlockBody
-    proposer_index: ValidatorIndex
-    whisk_opening_proof: ByteList[WHISK_MAX_OPENING_PROOF_SIZE]  # [New in Whisk]
```

Comment:
Best to add new fields in body and not in BeaconBlock obj, such that the BeaconBlockHeader maps to BeaconBlock with body as root